.travis.yml

```diff
@@ -10,18 +10,18 @@
 
 language: node_js
 os:
+  - windows
   - linux
   - osx
 
 node_js:
   - "10"
-  - "10.15"
-  - "10.19"
```

Comment:
These versions should be left in there as we want to explicitly test it in the node js versions this code runs in Runtime.